### PR TITLE
Adds Photojournalist profession

### PR DESCRIPTION
**Pull request by 2birdie**
Adds a photojournalist profession, starts with a Frank West inspired outfit and a camera pro.
A cool dude helped me pick out underwear.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
Scooped ya by a few months, although your description is best:
```
{
    "type": "profession",
    "ident": "photojournalist",
    "name": "Photojournalist",
        "description": "You were covering the breakup of society.",
        "points": 2,
        "skills": [
            {
                "level": 2,
                "name": "fabrication"
            },
            {
                "level": 2,
                "name": "speech"
            }
        ],
        "items": {
            "both": [
                "balclava",
                "longshirt",
                "jacket_light",
                "vest",
                "runner_bag",
                "gloves_light",
                "pants",
                "socks_wool",
                "sneakers",
                "eink_tablet_pc",
                "camera_pro",
                "cell_phone",
                "mobile_memory_card_used",
                "mobile_memory_card_used",
                "mobile_memory_card",
                "battery",
                "mask_dust"
         ],
            "male": [
                "briefs"
            ],
            "female": [
                "bra",
                "panties"
            ]
        },        "addictions": [
            {
                "intensity": 20,
                "type": "alcohol"
            }
        ]
    },
    
```
The game still needs a good photography mechanic, but I don't have the skill to implement it.

File: data/json/professions.json [hunk 1]
```diff
@@ -4599,6 +4599,30 @@
     },
     {
         "type": "profession",
+        "ident": "photojournalist",
+        "name": "Photojournalist",
```
Comment (Rivet-the-Zombie) [inline data/json/professions.json]: Double space after sentence breaks, please.

File: data/json/professions.json [hunk 2]
```diff
@@ -4599,6 +4599,30 @@
...
+        "description": "You were a freelance photojournalist before the end. You have a chance to be the first journalist to cover the apocalypse, though finding a publisher seems more difficult a prospect than usual. You managed to hold onto your camera, hopefully you can get some fantastic shots.",
```
Comment (Rivet-the-Zombie) [inline data/json/professions.json]: Double space after sentence breaks, please.

**Comment by 2birdie** [comment]
Oh. :P Well done, and thank you.

**Comment by 2birdie** [comment]
Forgot about that. Thanks, Rivet.
Should I keep this open, Bork? I'll concede the profession to you if you'd like, you had the idea first. I really am fine with that.

**Comment by mugling** [comment]
Anyone object to implementing a `More professions` mod to contain this and other extra professions? Particularly (un)-popular professions could then be moved between the mod and core as appropriate.

**Comment by BorkBorkGoesTheCode** [comment]
> Forgot about that. Thanks, Rivet.
 Should I keep this open, Bork? I'll concede the profession to you if you'd like, you had the idea first. I really am fine with that.

Both professions could coexist unchanged just like the several military professions. I'll change the name of my profession

**Comment by Rivet-the-Zombie** [comment]
>a `More professions` mod

If they were something thematically inappropriate, sure. Stuff like this one are fine the way they are.

**Pull request merged**